src/SUMMARY.md

```diff
@@ -10,9 +10,9 @@
 - [Uitproberen, installeren en starten](./installatie/README.md)
   - [Uitproberen](./installatie/uitproberen.md)
   - [Installeren en starten](./installatie/installeren-starten/README.md)
-    - [Methode 1: installatieprogramma downloaden en installeren (Windows)](./installatie/installeren-starten/methode-1.md)
-    - [Methode 2: installatiebestand downloaden en starten met systemd (Linux)](./installatie/installeren-starten/methode-2.md)
-    - [Methode 3: Docker Compose (Linux, macOS, Windows)](./installatie/installeren-starten/methode-3.md)
+    - [Installatieprogramma Windows](./installatie/installeren-starten/methode-1.md)
+    - [Installatiebestand Linux](./installatie/installeren-starten/methode-2.md)
+    - [Development met Docker Compose](./installatie/installeren-starten/methode-3.md)
```

Comment:
Moeten we dit nog als installatiemethode beschrijven? Of hoort deze eerder onder een hoofdstukje als 'Meewerken als ontwikkelaar?'